### PR TITLE
Fixed the counting of mb / gb counters

### PR DESCRIPTION
**Pull request by etorreborre**
@raronson @domdere

### CODE REVIEW

File: notion-distcopy/src/main/scala/com/ambiata/notion/distcopy/DistCopyJob.scala [hunk 1]
```diff
@@ -187,8 +189,9 @@ class DistCopyMapper extends Mapper[NullWritable, Mapping, NullWritable, NullWri
             } else {
               println(s"\tRunning stream upload")
               totalBytesUploaded.increment(length)
-              totalMegaBytesUploaded.increment(length / 1024 / 1024)
-              totalGigaBytesUploaded.increment(length / 1024 / 1024 / 1024)
+              val bytes = totalBytesUploaded.getValue
+              totalMegaBytesUploaded.increment(bytes / 1024 / 1024)
+              totalGigaBytesUploaded.increment(bytes / 1024 / 1024 / 1024)
```
Comment (raronson) [inline notion-distcopy/src/main/scala/com/ambiata/notion/distcopy/DistCopyJob.scala]: This is still `increment`

**Comment by raronson** [comment]
All the changes look the same, can you move it into a function?

**Comment by etorreborre** [comment]
@raronson take 357 is ready for review

File: notion-distcopy/src/main/scala/com/ambiata/notion/distcopy/DistCopyJob.scala [hunk 2]
```diff
@@ -204,8 +196,13 @@ class DistCopyMapper extends Mapper[NullWritable, Mapping, NullWritable, NullWri
       case Ok(_) =>
         ()
     }
+  }
 
-
+  def incrementCounters(bytesCounter: Counter, mbCounter: Counter, gbCounter: Counter, increment: Long) = {
```
Comment (raronson) [inline notion-distcopy/src/main/scala/com/ambiata/notion/distcopy/DistCopyJob.scala]: return type /nit

**Comment by raronson** [comment]
:+1:

**Pull request merged**